user-card/src/components/UserList.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import axios from "axios";
+import UserCard from "./UserCard";
+
+export default class UserList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: []
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("https://api.github.com/users/john-afolabi/followers")
+      .then(response => {
+        response.data.forEach(user => {
+          axios
+            .get(`${user.url}`)
+            .then(response => {
+              this.setState({
+                users: [...this.state.users, response.data]
+              });
+            })
+            .catch(error => {
+              alert(error);
+            });
+        });
+      })
+      .catch(error => {
+        alert(error);
```

Comment:
`error` is an object, in the event the code gets to execute, it'd throw an error, you could do `alert(error.message())`